apps/emqx_dashboard/src/emqx_dashboard_monitor.erl

```diff
@@ -256,8 +261,16 @@ merge_cluster_sampler_map(M1, M2) ->
 merge_cluster_rate(Node, Cluster) ->
     Fun =
         fun
-            (topics, Value, NCluster) ->
-                NCluster#{topics => Value};
+            %% cluster-synced values
+            (topics, V, NCluster) ->
+                NCluster#{topics => V};
+            (retained_msg_count, V, NCluster) ->
+                NCluster#{retained_msg_count => V};
+            (license_quota, V, NCluster) ->
+                NCluster#{license_quota => V};
+            %% for cluster sample, ignore node_uptime
+            (node_uptime, _V, NCluster) ->
+                NCluster;
```

Comment:
Maybe include this uptime in the Prometheus metrics as well. It would be very useful!